cpp/ql/lib/semmle/code/cpp/security/FunctionWithWrappers.qll

```diff
@@ -154,7 +154,7 @@ abstract class FunctionWithWrappers extends Function {
    * Whether 'arg' is an argument in a call to an outermost wrapper function of 'this' function.
    */
   predicate outermostWrapperFunctionCall(Expr arg, string callChain) {
-    exists(Function targetFunc, Call call, int argIndex |
+    exists(Function targetFunc, FunctionCall call, int argIndex |
```

Comment:
Same as above.

Reply:
Fixed in 8b953e4f22bdb8adfbce6bae64702eaef5328443